sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Release History
 
-## 3.0.0-beta.1 (Unreleased)
-First stable release of the azure-ai-formrecognizer package. Targets the service's v2.0 API.
+## 3.0.0-beta.1 (2020-08-11)
```

Comment:
Because of this https://teams.microsoft.com/l/message/19:d6ff4003f5c848a2a2d6dcdfc0ebd497@thread.skype/1595548814681?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1595548814681&teamName=Azure%20SDK&channelName=Service%20-%20Text%20Analytics&createdTime=1595548814681

basically applying this strategy for our track 2 library releases 
>The maximum released version of a Track 1 library across all languages +1

Reply:
In other libraries we've added some explanation in the changelog about this. like
> - Updated version number from `1.0.0-preview.5` to `3.0.0-preview.1`.

Consider adding it too

Reply:
Agree. Add a small note on why we changed major version from 1 to 3.